### DISCUSSION TITLE
GDPR Google Fonts

### DISCUSSION THREAD

**Discussion by ToshY** [tarampampam/error-pages#129] [General]
After finding out that that serving fonts through Google Fonts is not GDPR compliant ([bunny.net](https://bunny.net/blog/bringing-privacy-back-into-your-own-hands-introducing-bunny-fonts/); [official ruling](https://rewis.io/urteile/urteil/lhm-20-01-2022-3-o-1749320/)), is it maybe an idea to update this in the templates by switching to another font service, like for example [Bunny Fonts](https://fonts.bunny.net/about)? It would be as easy as just replacing `fonts.googleapis.com` by `fonts.bunny.net`.

**Comment by tarampampam** [comment]
Yes, sure!

I didn't think of that at all... thank you! Could you please make a PR with the update?

**Reply by tarampampam** [reply]
Or maybe use smt like a jsdevr, what do u think? 🤔

**Reply by ToshY** [reply]
> I didn't think of that at all... thank you! Could you please make a PR with update?

Yeah sure I can make a PR!

> Or maybe use smt like a jsdevr, what do u think?

I'm not sure if jsDelivr can be an easy stand-in replacement for this as I don't have (much) experience with it myself. How would you do this for example for the `Roboto` then?

I thought a simple search-and-replace with the `bunny.net` domain would be the easiest thing to do without to much hassle, but if you know an easy way to this with jsDelivr I'm okay with that too 😄

**Reply by tarampampam** [reply]
Ok, let's use the `bunny.net` 😄

**Reply by ToshY** [reply]
Should I create a corresponding issue related to this discussion or just submit the PR directly?

**Reply by pimjansen** [reply]
Isnt it a better idea to embed all the stuff instead of depending on a third party service again? Google going down not so much but bunny.net not so sure. Meaning that if their network is having latency issues you will notice that on your internal infrastructure.

So maybe it is just as easy or to embed or to just use a global available font family instead

**Reply by ToshY** [reply]
> Isnt it a better idea to embed all the stuff instead of depending on a third party service again? Google going down not so much but bunny.net not so sure. Meaning that if their network is having latency issues you will notice that on your internal infrastructure.
> 
> So maybe it is just as easy or to embed or to just use a global available font family instead

The idea was just to change to a reliable GDPR compliant party as a stand-in replacement for Google Fonts. Embedding/self-hosting in seems like relatively more effort/complicated than it actually needs to be.

@tarampampam what's your view on this?

**Reply by tarampampam** [reply]
Agreed with @ToshY at this moment. Maybe later I'll take care of the resources embedding 👍🏼 

The issue is optional, to your choice 😄

**Reply by pimjansen** [reply]
> Agreed with @ToshY at this moment. Maybe later I'll take care of the resources embedding 👍🏼 
> 
> 
> 
> The issue is optional, to your choice 😄 

Agree! Ppl can always create a custom container for their needs and use yours as reference.

**Reply by nickelswitte** [reply]
Similarly to @ToshY , I would also vote for fully embedded fonts and graphics. From a GDPR compliance and also efficiency stand-point, fully local resouces are preferable.

**Reply by tarampampam** [reply]
Each template included in v3 no longer contains external resources. Can this question be considered closed now? 😸

**Reply by lnxbil** [reply]
> Each template included in v3 no longer contains external resources. Can this question be considered closed now? 😸

Is that the reason the matrix theme is no longer present? I did not find information about the removal, just the note that it has been removed.
